services/api-server/src/simcore_service_api_server/utils/http_calls_capture_processing.py

```diff
@@ -0,0 +1,257 @@
+from pathlib import Path
+from typing import Any, Literal, get_args
+from urllib.parse import unquote
+
+import httpx
+import jsonref
+from pydantic import BaseModel, Field, parse_obj_as, root_validator, validator
+from simcore_service_api_server.core.settings import CatalogSettings, StorageSettings
+
+service_hosts = Literal["storage", "catalog"]
+
+
+class ParamSchema(BaseModel):
+    title: str | None
+    param_type: Literal["str", "int", "float", "bool"] | None = Field(
```

Comment:
TIP: when the name collides with a built-in word, I tend to name it as close as possible as
```python

class A(BaseModel):
    id_ : int = Field(..., alias="id")
    type_ : int = Field(..., alias="type")
```

It is easier to remember

Reply:
Nice tip. Fixed